CONTRIBUTING.rst

```diff
@@ -0,0 +1,165 @@
+Feedback and Contribution
```

Comment:
Here I think it is better to just have a link to the CONTRIBUTING file in the main repository, so we don't have to worry about keeping these files in sync.

Reply:
There are some different small details between both files, like URL links, repo names.
Do you think it could be a problem?

Reply:
I don't think it will be a problem, especially considering that people working with ross-ml will be users that are more used to work with ROSS.

Reply:
Done.